web/src/lib/components/admin-page/settings/machine-learning-settings/machine-learning-settings.svelte

```diff
@@ -77,6 +77,40 @@
         </div>
       </SettingAccordion>
 
+      <SettingAccordion
+        key="duplicate-detection"
+        title="Duplicate Detection"
+        subtitle="Use CLIP embeddings to find very similar images that are likely duplicates"
+      >
+        <div class="ml-4 mt-4 flex flex-col gap-4">
+          <SettingSwitch
+            id="enable-duplicate-detection"
+            title="ENABLED"
+            subtitle="If disabled, images will not be checked to see if they are duplicates."
+            bind:checked={config.machineLearning.duplicateDetection.enabled}
+            disabled={disabled || !config.machineLearning.enabled || !config.machineLearning.clip.enabled}
+          />
+
+          <hr />
+
+          <SettingInputField
+            inputType={SettingInputFieldType.NUMBER}
+            label="MAX DETECTION DISTANCE"
+            bind:value={config.machineLearning.duplicateDetection.maxDistance}
+            step="0.01"
+            min={0.001}
```

Comment:
I would agree, but @mertalev said not to when we lowered the minimum on the backend and UI.

Reply:
I plan to test, fine-tune, and enable this feature by default in the subsequent PR. THis PR is dedicated mainly to UI work

Reply:
I was originally against this because 0.001 increments are too subtle unless you're at the very bottom of the range and add more clicks. But it's fine since you can hold the arrow to move it quickly anyway.

Reply:
```suggestion
            step="0.001"
            min={0.001}
```
I think this change will do it.

Reply:
> I was originally against this because 0.001 increments are too subtle unless you're at the very bottom of the range and add more clicks. But it's fine since you can hold the arrow to move it quickly anyway.

I've been thinking about this as well, but there are also options in between .001 and .01. Maybe something like .005 could also be feasible? Ultimately it probably comes down to testing different options though

Reply:
@alextran1502 you want to stick with 0.01 for now?

Reply:
@danieldietzler yes, I can modify it in subsequent PR after fine-tuning the value